tutorial/app_is_leaking_memory_eventual_crash.md

```diff
@@ -0,0 +1,94 @@
+# App is leaking memory (eventual crash/hang)
+
+http://localhost:5000/api/diagscenario/memleak/{kb}
+
+In this scenario, the endpoint will slowly start leaking memory (amount specified by {kb}) and eventually will result in an out of memory exception. In order to diagnose this scenario, we need several key pieces of diagnostics data.
+
+### Memory counters
+Before we dig into collecting diagnostics data to help us root cause this scenario, we need to convince ourselves that what we are actually seeing is a memory leak (memory growth). On Windows we could use the myriad of .NET performance counters, but what about on Linux? It turns out .net core has been instrumented to expose metrics from the runtime and we can use the dotnet-counters tool to get at this information (please see 'Installing the diagnostics tools' section). 
+
+Next, lets run the webapi (dotnet run) and before hitting the above URL that will cause the leak, lets check our managed memory counters:
+
+> ```bash
+> dotnet-counters monitor gc-heap-size --refresh-interval 1 -p 4807
+> ```
+
+4807 is the process identifier which can be found using dotnet-trace list-processes. The refresh-interval is the number of seconds before refreshes. 
```

Comment:
dotnet-trace list-process should be `dotnet trace list-process`.

Reply:
pretty sure `dotnet-trace list-process` still works.

Reply:
`dotnet-trace` is better given that it won't try searching for a project file, although your suggestion works as well. However @MarioHewardt can we escape the comments in between back-ticks? Makes it easier to read and differentiate from the regular text.

Reply:
Just fyi, the plan is that as soon as we move this to the tutorial folder I'm going to merge it and @MarioHewardt will keep iterating on it. Scrutinizing at this level of detail probably isn't necessary yet : )